kartotherian/Dockerfile

```diff
@@ -25,19 +26,21 @@ RUN apt-get update && apt-get install -y nodejs \
     libmapbox-variant-dev \
     libboost-program-options-dev \
     libboost-thread-dev \
-    apt-transport-https \
-    libmapnik3.0 \
+    libmapnik3.1 \
     libmapnik-dev \
     mapnik-utils \
     mapnik-doc \
     && rm -rf /var/lib/apt/lists/*
 
+# Explicitly set $HOME variable to avoid an issue with nvm setup
+ENV HOME /root
```

Comment:
I don't think its a good idea to use `/root` and then change permissions. It would create potentially more confusions in the future. Maybe its better to use:

* User id  passed as an env var (or .env line) which defaults to 1000
* Create a user with that ID (eg. kartotherian or kartodock) and with a home folder
* Use that for mounting the codebase

Reply:
We're running into a different issue with node as root.  The `--unsafe-perm` flag allows the top-level `npm install` to succeed, but `lerna bootstrap` then fails because when it runs npm install in subdirectories, the flag isn't carried along.

Ideally we would be running npm as a non-root user, I think?